XXX crashes gnome-shell with:
glibtop(c=57582): [ERROR] glibtop_get_swap: Function not implemented
gnome-session-binary[34511]: WARNING: Application 'org.gnome.Shell.desktop' killed by signal 6
#2  0x000003b7b576f2fb in _libc_abort () at /usr/src/lib/libc/stdlib/abort.c:51
#3  0x000003b730cc8b7f in glibtop_error_io_vr () from /usr/local/lib/libgtop-2.0.so.10.1
#4  0x000003b730cc8d99 in glibtop_error_io_r () from /usr/local/lib/libgtop-2.0.so.10.1
#5  0x000003b730cc61bd in glibtop_get_swap_l () from /usr/local/lib/libgtop-2.0.so.10.1

https://gitlab.gnome.org/GNOME/libgtop/-/issues/44

Index: meson.build
--- meson.build.orig
+++ meson.build
@@ -43,7 +43,6 @@ default_extensions += [
   'light-style',
   'screenshot-window-sizer',
   'status-icons',
-  'system-monitor',
   'windowsNavigator',
   'workspace-indicator'
 ]
